x-pack/test/api_integration/apis/search/session.ts

```diff
@@ -48,8 +48,6 @@ export default function ({ getService }: FtrProviderContext) {
           .expect(200);
 
         await supertest.delete(`/internal/session/${sessionId}`).set('kbn-xsrf', 'foo').expect(200);
```

Comment:
I think we need to update the title of this test if the session is not actually deleted.
We could also validate that after re-fetching the session, the status is updated correctly.